.github/workflows/ci.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       - name: Filter directories
```

Comment:
*arbitrary comment location*

Just starting a review here @alexkeating but I wanted to let you know I got the following test failure running these locally:

```
Running 1 test for test/WormholeL1ERC20Bridge.t.sol:ReceiveWormholeMessages
[FAIL. Reason: Expected an emit, but the call reverted instead. Ensure you're testing the happy path when using the `expectEmit` cheatcode. Counterexample: calldata=0x0x8146fd3b000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000, args=[0, 0, 0]] testFuzz_CastVoteOnL1(uint32,uint32,uint32) (runs: 0, μ: 0, ~: 0)
Traces:
  [353983] ReceiveWormholeMessages::testFuzz_CastVoteOnL1(0, 0, 0)
    ├─ [165872] FakeERC20::mint(ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], 79228162514264337593543950335 [7.922e28])
    │   ├─ emit Transfer(from: 0x0000000000000000000000000000000000000000, to: ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], value: 79228162514264337593543950335 [7.922e28])
    │   ├─ emit DelegateChanged(delegator: ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], fromDelegate: 0x0000000000000000000000000000000000000000, toDelegate: ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496])
    │   ├─ emit DelegateVotesChanged(delegate: ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], previousBalance: 0, newBalance: 79228162514264337593543950335 [7.922e28])
    │   └─ ← ()
    ├─ [24626] FakeERC20::approve(ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], 79228162514264337593543950335 [7.922e28])
    │   ├─ emit Approval(owner: ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], spender: ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], value: 79228162514264337593543950335 [7.922e28])
    │   └─ ← true
    ├─ [63992] FakeERC20::transferFrom(ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], WormholeL1ERC20Bridge: [0xc7183455a4C133Ae270771860664b6B7ec320bB1], 79228162514264337593543950335 [7.922e28])
    │   ├─ emit Approval(owner: ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], spender: ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], value: 0)
    │   ├─ emit Transfer(from: ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], to: WormholeL1ERC20Bridge: [0xc7183455a4C133Ae270771860664b6B7ec320bB1], value: 79228162514264337593543950335 [7.922e28])
    │   ├─ emit DelegateVotesChanged(delegate: ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], previousBalance: 79228162514264337593543950335 [7.922e28], newBalance: 0)
    │   ├─ emit DelegateVotesChanged(delegate: WormholeL1ERC20Bridge: [0xc7183455a4C133Ae270771860664b6B7ec320bB1], previousBalance: 0, newBalance: 79228162514264337593543950335 [7.922e28])
    │   └─ ← true
    ├─ [60539] GovernorFlexibleVotingMock::propose([0xF62849F9A0B5Bf2913b396098F7c7019b51A820a], [0], [0x40c10f19000000000000000000000000000000000000000000000000000000000000000000000000000000005991a2df15a8f6a256d3ec51e99254cd3fb576a900000000000000000000000000000000000000000000000000000000000186a0], Proposal: To inflate token)
    │   ├─ [1393] FakeERC20::getPriorVotes(ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], 26662437 [2.666e7]) [staticcall]
    │   │   └─ ← 0
    │   ├─ emit ProposalCreated(proposalId: 453455188748263905522686351817970882667777794382561448165884150873180888254 [4.534e74], proposer: ReceiveWormholeMessages: [0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496], targets: [0xF62849F9A0B5Bf2913b396098F7c7019b51A820a], values: [0], signatures: [], calldatas: [0x40c10f19000000000000000000000000000000000000000000000000000000000000000000000000000000005991a2df15a8f6a256d3ec51e99254cd3fb576a900000000000000000000000000000000000000000000000000000000000186a0], voteStart: 26662442 [2.666e7], voteEnd: 26662458 [2.666e7], description: Proposal: To inflate token)
    │   └─ ← 453455188748263905522686351817970882667777794382561448165884150873180888254 [4.534e74]
    ├─ [649] GovernorFlexibleVotingMock::proposalDeadline(453455188748263905522686351817970882667777794382561448165884150873180888254 [4.534e74]) [staticcall]
    │   └─ ← 26662458 [2.666e7]
    ├─ [0] VM::roll(26662457 [2.666e7])
    │   └─ ← ()
    ├─ [2319] WormholeL1ERC20Bridge::owner() [staticcall]
    │   └─ ← DefaultSender: [0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38]
    ├─ [0] VM::prank(DefaultSender: [0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38])
    │   └─ ← ()
    ├─ [24683] WormholeL1ERC20Bridge::setRegisteredSender(6, 0x000000000000000000000000eac5f0d4a9a45e1f9fdd0e7e2882e9f60e301156)
    │   ├─ emit RegisteredSenderSet(owner: DefaultSender: [0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38], sourceChain: 6, sourceAddress: 0x000000000000000000000000eac5f0d4a9a45e1f9fdd0e7e2882e9f60e301156)
    │   └─ ← ()
    ├─ [0] VM::expectEmit()
    │   └─ ← ()
    ├─ emit VoteCast(voter: 0x0591C25ebd0580E0d4F27A82Fc2e24E7489CB5e0, proposalId: 453455188748263905522686351817970882667777794382561448165884150873180888254 [4.534e74], voteAgainst: 0, voteFor: 0, voteAbstain: 0)
    ├─ [0] VM::prank(0x0591C25ebd0580E0d4F27A82Fc2e24E7489CB5e0)
    │   └─ ← ()
    ├─ [1052] WormholeL1ERC20Bridge::receiveWormholeMessages(0x0100a583c06ce9658bd1fb4c44c031e00116b5b3fbbc8157f275a3cde0405cbe000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000, [], 0x000000000000000000000000eac5f0d4a9a45e1f9fdd0e7e2882e9f60e301156, 6, 0x0000000000000000000000000000000000000000000000000000000000000000)
    │   └─ ← "OnlyRelayerAllowed()"
    └─ ← "Log != expected log"

Test result: FAILED. 0 passed; 1 failed; 0 skipped; finished in 47.36ms
```

Reply:
I wasn't able to recreate this failure locally; even when I used the same input parameters. I did slightly modify the test. Let me know if it is still causing issues.